packages/module-loader-commonjs/src/__tests__/index.test.ts

```diff
@@ -52,4 +59,30 @@ describe('createCommonJsModuleLoader', () => {
 
     expect(loadedModule).toEqual(expectedModule);
   });
+
+  it('sets agent if process.env.HTTP_PROXY is defined', async () => {
+    const currentHttpProxy = process.env.HTTP_PROXY;
```

Comment:
I would prefer the name `initialHttpProxy`

Reply:
Hi @clebert, okay, I fixed it, thanks!